ibis/expr/rules.py

```diff
@@ -487,6 +493,62 @@ def python_literal(value, arg, **kwargs):
     return arg
 
 
+@functools.singledispatch
+def _make_hashable(value) -> Hashable:
```

Comment:
Do we still need `_make_hashable`?

Reply:
Nope, I thought I got rid of it. Doing it now.